crates/bevy_pbr/src/material.rs

```diff
@@ -490,7 +490,15 @@ impl<P: PhaseItem, M: Material, const I: usize> RenderCommand<P> for SetMaterial
     }
 }
 
-pub type RenderMaterialInstances<M> = ExtractedInstances<AssetId<M>>;
+/// Stores all extracted instances of a [`Material`] in the render world.
+#[derive(Resource, Deref, DerefMut)]
+pub struct RenderMaterialInstances<M: Material>(pub EntityHashMap<AssetId<M>>);
```

Comment:
Am I missing something? This doesn't seem to be used anywhere.

Reply:
You must be, it's being use in quite a few places :)

None of them show up in the diff because it's API is exactly what it was, in case that's where you were looking